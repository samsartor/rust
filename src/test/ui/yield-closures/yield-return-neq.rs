#![feature(yield_closures)]

fn main() {
    let _ = || {
        yield "";
        return 0; //~ ERROR: mismatched types
    };
}
