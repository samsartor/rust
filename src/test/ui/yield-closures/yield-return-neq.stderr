error[E0308]: mismatched types
  --> $DIR/yield-return-neq.rs:6:16
   |
LL |         return 0;
   |                ^ expected `&str`, found integer
   |
note: return type inferred to be `&str` here
  --> $DIR/yield-return-neq.rs:5:15
   |
LL |         yield "";
   |               ^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0308`.
